lib/trino-parquet/src/main/java/io/trino/parquet/reader/ColumnReaderFactory.java

```diff
@@ -200,7 +200,7 @@ public static ColumnReader create(PrimitiveField field, DateTimeZone timeZone, A
                 };
```

Comment:
is it tested already?

Reply:
Tests were already added in previous commit